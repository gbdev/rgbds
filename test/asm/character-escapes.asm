MACRO m
  DEF S EQUS \1
  println "{S}"
  println \1
  println "illegal character \
escape \z?"
  println "invalid character \<\n>?"
  println """invalid character \<	>?"""
ENDM
  m "(\n \r \t)"
