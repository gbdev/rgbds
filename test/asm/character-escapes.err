error: character-escapes.asm(10) -> character-escapes.asm::m(6):
    Illegal character escape 'z'
error: character-escapes.asm(10) -> character-escapes.asm::m(7):
    Invalid character in bracketed macro argument '\'
error: character-escapes.asm(10) -> character-escapes.asm::m(8):
    Invalid character in bracketed macro argument '\t'
error: Assembly aborted (3 errors)!
