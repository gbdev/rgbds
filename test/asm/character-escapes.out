(
  	)
(
  	)
illegal character escape z?
invalid character n>?
invalid character >?
